cpp/inc/bond/ext/grpc/client_callback.h

```diff
@@ -55,11 +55,6 @@ namespace bond { namespace ext { namespace gRPC {
               status(status),
               context(std::move(context))
         { }
-
```

Comment:
The (incorrect) intention behind these `= default` was to say that `unary_call_result`  is copyable and moveable. Can we add `static_assert` tests for these properties?

Reply:
Will add a test check. The `=default` is not for documenting the code, in my opinion.